ecowitt2mqtt/runtime.py

```diff
@@ -67,6 +67,15 @@ def __init__(self, ecowitt: Ecowitt) -> None:
             )
         )
 
+        @app.get("/health/liveness")
```

Comment:
Not necessarily something for this PR, but the original FastAPI app was designed to have one endpoint; if we're going to host more, we should abstract this out (i.e., `Runtime` should be explicitly aware).

Reply:
You could.. I don't really know enough about FastAPI. 
I dont really see you having more endpoints than this though unless you are supporting different pws (like how in my version i supported both ecowitt protocol and wunderground protocol). This is purely for the host infrastructure to be able to check the application is 'alive' and 'ready'. 

My gut would be do it this way for now and if you add another endpoint then refactor then as there is clearly a need for the abstraction then. I think doing the abstraction now for this would just add needless complexity to your code.